src/dashboard/Data/Browser/Browser.react.js

```diff
@@ -501,7 +500,7 @@ class Browser extends DashboardView {
           if (name === 'objectId' || this.state.isUnique && name !== this.state.uniqueField) {
             return;
           }
-          if (!!required) {
```

Comment:
Are these changes required for this PR?

Reply:
@mtrezza No, my IDE run your ESLint config through this file, sorry! Would you accept a PR with changes from running ESLint through the codebase?

Reply:
Normally I would be happy about lint changes, but this looks suspicious, because a double negation has a purpose.

Reply:
Thanks for reverting.